esphome/esphome.yaml

```diff
@@ -0,0 +1,64 @@
+substitutions:
+  name: "aiosense"  # lowercase only
+  # bh1750_update_interval: 5s
+
+  # bmex80_offset_humidity: "0"
+  # bmex80_offset_temperature: "0"
+  # bmex80_update_interval: 30s
+
+  # bme680_bsec_sample_rate: lp # lp -> 3s, ulp ->5min
+  # bme680_bsec_temperature_offset: "0"
+
+  # ld2410_max_distance_move: 4.5m
+  # ld2410_max_distance_still: 4.5m
+  # ld2410_timeout: 5s
+
+  # pir_delayed_off: 1s
+  # pir_delayed_on: 0s
+
+packages:
+  remote_package:
+    url: https://github.com/schluggi/AIOsense
+    ref: packages
```

Comment:
Der ref muss vor dem merge noch raus

Reply:
Yes. But otherwise no one could test it yet :)